### ISSUE TITLE
Not working in V12 stable

### ISSUE THREAD

**Issue by flynnkd** [schlosrat/token-health#85]
Module appears in list of available, but has no settings options (?). Does not do anything on token selection and activation.

**Comment by Bryan-Legend** [comment]
Here's a call stack of the error that is thrown in V12:

<img width="411" alt="image" src="https://github.com/schlosrat/token-health/assets/920120/06fe3eed-a010-40fb-ac43-29de6fdec3aa">

**Comment by Bryan-Legend** [comment]
Another error on startup.

<img width="405" alt="image" src="https://github.com/schlosrat/token-health/assets/920120/c271db52-ba9c-445a-8ce9-7749588fbd65">

I think this could be causing the settings to be null, causing the other problem.

**Comment by rmasoni** [comment]
This is hands-down one of the best Foundry modules out there. We're lucky it worked without issues on V11 despite being stuck in V10 for a long time.

We need someone to take up the torch.

**Comment by Bryan-Legend** [comment]
@schlosrat Would you be interested in handing the project off to someone else?

**Comment by schlosrat** [comment]
Yes. I've not used FVTT in some time, v10 I think. Happy to hand this off to anyone who would like to pick it up

**Comment by Bryan-Legend** [comment]
I can't take it over myself, but I posted to the league of extraordinary developers discord.

Apparently there's now a competing project at https://github.com/mclemente/fvtt-token-health

It was the result of a commission: https://apoapostolov.notion.site/Token-Health-Commission-914ab651b5804340b214a0de9e03e350
